Sources/ParseSwift/API/URLSession+extensions.swift

```diff
@@ -66,6 +66,10 @@ extension URLSession {
                           let json = try? JSONSerialization
                             .data(withJSONObject: responseData,
                               options: .prettyPrinted) else {
+                        let err = (error as NSError)
```

Comment:
Shorthand variable names like `err` are not the swift way and we don't use this type of naming in the SDK. You should make the name change along with other changes:

```swift
let nsError = error as NSError
if nsError.code == 4865,
  let description = nsError.userInfo["NSDebugDescription"] {
   return .failure(ParseError(code: .invalidStruct, message: "Invalid struct: \(description)"))
}
```